jsapp/js/components/multiCheckbox.es6

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import autoBind from 'react-autobind';
+import Checkbox from 'js/components/common/checkbox';
+
+/**
+ * @namespace MultiCheckboxItem
+ * @prop {boolean} checked
+ * @prop {boolean} disabled
+ * @prop {string} label
+ */
+
+/**
+ * A MultiCheckbox generic component.
+ *
+ * @prop {MultiCheckboxItem[]} items
+ * @prop {function} onChange - returns whole list whenever any item changes
+ */
+class MultiCheckbox extends React.Component {
+  constructor(props){
+    super(props);
+    autoBind(this);
+  }
+
+  onChange(response) {
+    let updatedList = this.props.items;
+    updatedList[response].checked = !updatedList[response].checked;
+    this.props.onChange(updatedList);
+  }
+
+  render() {
+    return (
+      <div className='project-downloads__column project-downloads__column--right'>
+        <span className='project-downloads__title'>
+          {this.props.label}
+        </span>
+        <ul className='project-downloads__questions-list'>
```

Comment:
We need these class names to be something specific, like `multi-checkbox`, `multi-checkbox__list`, `multi-checkbox__list-item`, for sure not reusing the `project-downloads` ones here.

Reply:
I think you can add new `.scss` file for `multi-checkbox`, don't mind it will be so small :)